service/server.go

```diff
@@ -2,16 +2,20 @@ package main
 
 import (
 	"flag"
+	"net/http"
+	"strings"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/gorilla/mux"
+	lru "github.com/hashicorp/golang-lru"
 	"github.com/intuit/katlas/service/apis"
 	"github.com/intuit/katlas/service/cfg"
 	"github.com/intuit/katlas/service/db"
 	"github.com/intuit/katlas/service/resources"
-	"net/http"
-	"strings"
 )
 
+const CacheSize = 10
```

Comment:
Please change first letter to lower case if it not used by others

Reply:
Made the change and committed.